### ISSUE TITLE
Resizing Issue

### ISSUE THREAD

**Issue by singhgulshan** [Mobius1/Pageable#3]
After resizing the browser window when again maximize the window. it remains in same size with scroll bar.
![screenshot from 2018-10-25 16-47-36](https://user-images.githubusercontent.com/36273489/47496846-bdaffa00-d875-11e8-8eba-e1fa8676a15b.png)

**Comment by Mobius1** [comment]
Need your OS and Chrome version so I can test.

**Comment by singhgulshan** [comment]
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.5 LTS
Release:	16.04
Codename:	xenial

Chrome: Version 70.0.3538.77 (Official Build) (64-bit)

**Comment by Mobius1** [comment]
I only have access to Windows, iOS and Centos 7 so I can't test on Ubuntu. Is the "update" event flashing for you on the playground page when resizing? Are there any errors in Chrome's console?

**Comment by singhgulshan** [comment]
Yes update event is flashing and there are no errors in chrome console

**Comment by Mobius1** [comment]
It's strange that opening the console is triggering the resize, but not resizing the window itself. I'll have to see if I can install Ubuntu on one of my servers and do some testing.

**Comment by dvlpr91** [comment]
Me too!
I'm using Chrome version 71 and Windows 10,

```
'window.addEventListener ("resize", this.callbacks.update, false);

Pageable.prototype.update = function () {
  ...
};
```

**Comment by GiorgosK** [comment]
Running `window.dispatchEvent(new Event('resize'));` in the console indeed does the trick
